runtime/tests/checker/entitlements_test.go

```diff
@@ -128,9 +129,10 @@ func TestCheckBasicEntitlementMappingNonEntitlements(t *testing.T) {
 			}
 		`)
 
-		errs := RequireCheckerErrors(t, err, 1)
+		errs := RequireCheckerErrors(t, err, 2)
 
-		require.IsType(t, &sema.InvalidNonEntitlementTypeInMapError{}, errs[0])
+		require.IsType(t, &sema.NotDeclaredError{}, errs[0])
```

Comment:
Basically because the entitlement mappings have to be processed by the checker before struct definitions (so they can be used in struct definitions), meaning that if someone puts a struct type inside an entitlement mapping (which doesn't work anyways) the checker doesn't know about that struct yet.